game-app/map-data/src/main/java/org/triplea/map/game/notes/GameNotes.java

```diff
@@ -19,10 +19,10 @@ public class GameNotes {
   public static String loadGameNotes(final Path xmlGameFile) {
     Preconditions.checkArgument(
         Files.exists(xmlGameFile),
-        "Error, expected file did not exist: " + xmlGameFile.toAbsolutePath());
+        "File Existantial Error, expected file did not exist: " + xmlGameFile.toAbsolutePath());
```

Comment:
Is "Existantial" even a real word? I'd use just "File Error" here, or drop the error prefix entirely so the error message becomes "Expected file did not exist: <filename>"

Reply:
I love this, an existential error for a file : )

ex·is·ten·tial
/ˌeɡzəˈsten(t)SHəl/

adjective
relating to existence.
"the climate crisis is an existential threat to the world"

PHILOSOPHY
concerned with existence, especially human existence as viewed in the theories of existentialism.
"the existential dilemma is this: because we are free, we are also inherently responsible"

LOGIC
(of a proposition) affirming or implying the existence of a thing.

-----------------

On a more serious note, error messages in TripleA are intended for users. The error message should as best as possible describe:
- what happened, what was the error
- what is the impact of the error. In this case, game notes cannot be loaded. Though, the way the error is handled we are going to crash the whole application. It could be appropriate to return an empty string and display a warning pop-up message to the user that no game notes could be loaded. Since the game crashes right after we show this error, there is not much need to describe impact because the application will have just crashed (impact is pretty obvious)
- last, what a user can do to fix the error, or what can the user check to help resolve the error. In this case simply stating that we expected a game notes file at a given path is pretty sufficient.

So, this error message probably could only be slightly improved to be more like:
```
String.format("Could not find game notes file at path: %s", xmlGAmeFile.toAbsolutePath());
```

If we update the code to return an empty string in this condition instead of throwing an exception, then an error message like this would be better:
```
String.format("Could not find game notes file. Game notes will not appear. Expected game notes file to be at location: %s, but could not find a file there. Contact the map maker by posting on forums to have a game notes file be created for this map.");
```